docs/python/run.md

```diff
@@ -0,0 +1,56 @@
+---
+Order: 3
+Area: python
+TOCTitle: Run Python Code
+ContentId:
+PageTitle: Running Python Code in Visual Studio Code
+DateApproved: 09/24/2024
+MetaDescription: Running Python Code in Visual Studio Code
+MetaSocialImage: images/tutorial/python-social.png
+---
+
+# Running Python in Visual Studio Code
+
+Whether you are experimenting with smaller lines of code in a REPL or ready to run a Python script, the Python extension offers a number of ways to run Python code.
+
+## Interactively running Python code
+
+The Python interpreter installed on your machine allows for what’s known as an interactive REPL (Read-Evaluate-Print Loop), which reads a piece of code, evaluates it, and then prints the result to the console. After installing a Python interpreter on your machine, you can interact with the Python REPL by searching for the Terminal / Command Prompt on you system, and typing `python` (Windows) or `python3` (macOS/Linux) to activate the Python REPL, notated by `>>>`. There are two additional ways you can interact with a Python REPL in VS Code.
+
+### Native REPL
+
+The VS Code Native REPL for Python builds upon the classic Python REPL and provides additional features such as Intellisense and syntax highlighting to make your Python development experience more efficient. However, this REPL still adheres to principles present in the REPL built-in to Python itself, in that historical execution order and its content are immutable.
+
+You can open the Native REPL via the Command Palette (`kb(workbench.action.showCommands)`) by searching for **Python: Start Native REPL**. Furthermore, you can send code to the Native REPL via Smart Send (`kbstyle(Shift+Enter)`) and **Run Selection/Line** by setting `"python.REPL.sendToNativeREPL": true` in your `settings.json` file. You can opt to continue to use the REPL built-in to Python located in the terminal ( `>>>` ) by setting `"python.REPL.sendToNativeREPL": false` in your `settings.json`.
```

Comment:
Similar to how **Run Selection/Line in Python Terminal** is put for Terminal REPL. We could change **Run Selection/Line** into **Run Selection/Line in Python REPL**